### DISCUSSION TITLE
How can i pass an event(click) inside a foreach loop

### DISCUSSION THREAD

**Discussion by josephDev123** [livewire/livewire#4297] [1. Help] [answered]
`@foreach ($todo as $item)
<div class="d-flex justify-content-between align-items-center w-75 mx-auto" wire:key='{{ $item->id }}'>
  <div>
    <h4>{{ $item->content }} </h4>
  </div> 
  
  <div>
      
    <button type="button" class="btn btn-primary mb-2" data-bs-toggle="modal" data-bs-target="#Modal" wire:click="$emit('updateTodo', $item->id)"> Update </button>
    <button type="button" class="btn btn-danger mb-2" data-bs-toggle="modal" data-bs-target="#Modal"> Delete </button>
  </div>
</div>
@endforeach`

**Comment by iBotPeaches** [comment, marked as answer]
`wire:click="doSomething"` on whatever you are drawing out there to click. With `doSomething` being the method on the corresponding Livewire class.
